Black formatting preparation

clacroix12 opened the pull request:
This PR fixes various issues that were causing pre-commit to fail after formatting the repo with black. These failures were coming up now because pre-commit runs against files in your changeset and not against the entire repository. These problems existed before black formatting was being done.

mbukatov commented on ocs_ci/utility/gcp.py:
```diff
@@ -26,17 +26,16 @@
+# cluster access as well.
+#
+# For more details, see `GCP documentation on ServiceAccountKey resource
+# <https://cloud.google.com/iam/docs/reference/rest/v1/projects.serviceAccounts.keys>`_
```
Would this change preserve this docstring in our sphinx build?

Right now, it's part of our documentation as expected: https://ocs-ci.readthedocs.io/en/latest/apidoc/ocs_ci.utility.html#ocs_ci.utility.gcp.SERVICE_ACCOUNT_KEY_FILEPATH

clacroix12 replied:
Hmm, this I'm not sure of. Right now it's causing the following error in our `check-docstring-first` hook:
```
Check docstring is first.................................................Failed
- hook id: check-docstring-first
- exit code: 1

ocs_ci/utility/gcp.py:29 Multiple module docstrings (first docstring on line 2).
```

vasukulkarni replied:
I think we can check the docs from this build and resolve it if it looks good

clacroix12 replied:
There is actually an open issue for this https://github.com/pre-commit/pre-commit-hooks/issues/159 with a potential fix that has yet to be merged. It may be worth reconsidering the use of `check-docstring-first` until that is fixed if we want to use attribute docstrings.

clacroix12 replied:
Actually, looks like there may be a workaround for this: https://github.com/pre-commit/pre-commit-hooks/issues/159#issuecomment-559886109

mbukatov replied:
This is a bug in `check-docstring-first`, likely https://github.com/pre-commit/pre-commit-hooks/issues/159 - as you can see in the html sphinx build, both module docstring and variable docstring are present as expected.

I suggest to disable this particular hook.

clacroix12 replied:
Yeah, I have tried to use the `#:` syntax workaround but it doesn't seem to work for me. I'm not seeing any attribute doc unless I go with the triple quote docstring method.

> I suggest to disable this particular hook.
I agree, I am going to revert the change to this docstring and disable the check entirely.

clacroix12 commented:
@vasukulkarni @mbukatov I have reverted the change to that particular docstring and removed `check-docstring-first` from our pre-commit hooks. I don't think it offers enough for us to limit our documentation for it.
